fdbclient/Schemas.cpp

```diff
@@ -536,6 +536,7 @@ const KeyRef JSONSchemas::statusSchema = LiteralStringRef(R"statusSchema(
       "data_distribution_disabled_for_rebalance":true,
       "data_distribution_disabled":true,
       "configuration":{
+         "active_primary_dc":"pv",
```

Comment:
My one concern with putting this in the `configuration` section is that I think someone could decide to take this entire section, dump it into a file, and use `fdbcli` to set the configuration of the database using its contents. I see you've also updated `configurationSchema` to accept this field, but presumably its presence will not actually have any effect on how the database is configured, which might be confusing.

We could opt to move it out a level such that it sits at the same level as `configuration`, or maybe we could create a new section for multi-region state in anticipation of adding new fields in the future as the feature gets more sophisticated.

Reply:
Good point. I think we can simply remove this key from `configurationSchema` to make it clear that it's not a value that can be set by user.

Reply:
If we remove it, then I think it becomes more imperative that we also move it out of this section in status, as the copy to file workflow would break if this field were present.

Reply:
Ah I didn't know there is a workflow doing this. OK I will move it up a level.

Reply:
Done